takwimu/profiles/census.py

```diff
@@ -1985,17 +2054,17 @@ def get_profile(geo, profile_name, request):
                 func = globals()[function_name]
                 data[section] = func(geo, session, country, level)
                 # get profiles for comparative geometries
-                if not data[section]['is_missing'] and not data[section]['is_missing'] is None:
-                    for comp_geo in comparative_geos:
-                        try:
-                            merge_dicts(
-                                data[section], func(
-                                    comp_geo, session, country, level), comp_geo.geo_level)
-                        except KeyError as e:
-                            msg = "Error merging data into %s for section '%s' from %s: KeyError: %s" % (
-                                geo.geoid, section, comp_geo.geoid, e)
-                            log.fatal(msg, exc_info=e)
-                            raise ValueError(msg)
+                # if not data[section]['is_missing']):
```

Comment:
What's happening here?

Reply:
this is the`data comparison` code for geos and it's comparative geo (parent geography). It is throwing error for now as available data between two geos are not the same. I will remove the comment code all together and create a bug story to fix the throwing of error on wazimap.

We haven't been using it however